rllib/utils/test_utils.py

```diff
@@ -111,12 +117,24 @@ def framework_iterator(config=None,
             for tracing in [True, False]:
                 config["eager_tracing"] = tracing
                 print(f"framework={fw} (eager-tracing={tracing})")
+                time_started = time.time()
                 yield fw if session is False else (fw, sess)
+                if time_iterations is not None:
+                    time_total = time.time() - time_started
+                    time_iterations[fw + ("+tracing" if tracing else "")] = \
+                        time_total
+                    print(f".. took {time_total}sec")
```

Comment:
maybe put these 4 lines in a small local util _record_iter_time(fw, start, tracing) to cut down duplication?